docs/source/plugins/02_alfasim_sdk_by_example.rst

```diff
@@ -218,7 +218,7 @@ needed to load and unload the |sdk| API, in which will allows the plugin to use
 Then, since the plugin wants to calculate its own secondary variable, the
 :func:`HOOK_UPDATE_PLUGINS_SECONDARY_VARIABLES <alfasim_sdk._internal.hook_specs.update_plugins_secondary_variables>` must be implemented.
 As can be seen in the example below, to retrieve the velocity of the continuous liquid field
-it is necessary to use the :func:`get_simulation_array` API function.
+it is necessary to use the ``get_simulation_array`` API function.
```

Comment:
You are removing the link. You should only add `:cpp:` in front of it, like `:cpp:func:`.
In that way the link will be fixed.

Reply:
Nice, added!